### ISSUE TITLE
Should `yarn` be mentioned as a step during installation

### ISSUE THREAD

**Issue by kgrz** [facebookexperimental/rome#206]
Even though yarn/node_modules aren't strictly required for the base installation, the `engines` setting in `package.json` is not checked during `./script/build-release dist` (one of the steps in documentation) is run. I ran into an error while running this on node 8.9.4[1], which didn't point to this. Running `yarn` in the folder after clone would hint users to this better, making the UX better.

Should the step be added in docs?

[1]:

```
> rome-root@0.0.2 build /Users/kashyap.k/Documents/github_clones/rome
> ./scripts/build-release

 Building trunk
Error thrown inside the CLI handler
TypeError: stdout.off is not a function
    at Object.outStream.teardown (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:69854:20)
    at ___R$project$rome$$romejs$cli$reporter$Reporter_ts$default.removeStream (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:70010:16)
    at ___R$project$rome$$romejs$cli$reporter$Reporter_ts$default.teardown (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:70419:14)
    at endEvent.subscribe (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:94782:25)
    at ___R$project$rome$$romejs$events$Event_ts$default.call (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:5950:29)
    at ___R$project$rome$$romejs$events$Event_ts$default.callOptional (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:6022:21)
    at ___R$project$rome$$romejs$core$client$Client_ts$default.end (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:95097:27)
    at ___R$project$rome$$romejs$cli$cli_ts$default (/Users/kashyap.k/Documents/github_clones/rome/scripts/vendor/rome.cjs:98319:18)
    at <anonymous>
```

**Comment by sebmck** [comment]
I don't think it makes sense to recommend running `yarn` or `npm` just to get the `engines` validation. We could add a step to all `scripts` to check it instead.

**Comment by sebmck** [comment]
Just created and merged #209 which adds the message to all `scripts`. You will still encounter syntax errors if your Node version doesn't support any of the syntax that the scripts use, but that's a much better indicator for an outdated Node than some obscure API calls. Thanks for the report!
